spec/features/users/user_logs_in_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "user logs into account" do
+
+  let(:new_user) { User.create(email: "mckelvey.matt@gmail.com", password: "12345678") }
```

Comment:
Use a `let!(:new_user)` instead of just plain old let - that'll mean you don't need to call `new_user` on line 9! See my earlier comment on `let!` for an explanation